tests/bootstrap.php

```diff
@@ -89,6 +89,7 @@ function _manually_load_plugin() {
 	define( 'WC_USE_TRANSACTIONS', false );
 	update_option( 'woocommerce_enable_coupons', 'yes' );
 	update_option( 'woocommerce_calc_taxes', 'yes' );
+	define( 'WOOCOMMERCE_ADMIN_SHOW_ONBOARDING', true );
```

Comment:
Maybe we could rename this to `WOOCOMMERCE_ADMIN_ONBOARDING_ENABLED` to be slightly more accurate since this won't necessarily force it show at all times.